lms/security.py

```diff
@@ -99,11 +99,11 @@ def get_policy(request: Request):  # noqa: PLR0911
 
         path = request.path
 
-        if path.startswith("/admin") or path.startswith("/googleauth"):
+        if path.startswith(("/admin", "/googleauth", "/dashboard/org")):
```

Comment:
I think it wouldn't make a difference, but perhaps this should check for `/dashboard/orgs`, with an `s` at the end?

```suggestion
        if path.startswith(("/admin", "/googleauth", "/dashboard/orgs")):
```